cmd/openapi-generator-go/cmd/filter.go

```diff
@@ -56,6 +58,9 @@ var filterCmd = &cobra.Command{
 		spec := bytes.NewReader(bs)
 		filteredSpec, err := filters.ByPath(spec, allowedPaths)
 		if err != nil {
+			if zerolog.GlobalLevel() == zerolog.DebugLevel {
+				fmt.Println(string(filteredSpec))
```

Comment:
when this fails, it is super mysterious, this will now print the failing Go code to stdout so that you can re-run with debug enabled and see _why_ the mode is failing

Reply:
note that i put this to stdout out so that you can do something like

```
openapin-generator-go filter .... | pbcopy
```

and you get just the failing go model